src/govuk/components/notification-banner/_index.scss

```diff
@@ -0,0 +1,80 @@
+@include govuk-exports("govuk/component/notification-banner") {
+  .govuk-notification-banner {
+    @include govuk-font($size: 19);
+    @include govuk-responsive-margin(8, "bottom");
+
+    border: $govuk-border-width solid $govuk-brand-colour;
+
+    &:focus {
+      outline: $govuk-focus-width solid $govuk-focus-colour;
+    }
+  }
+
+  .govuk-notification-banner__header {
+    padding: 0 govuk-spacing(3) govuk-spacing(1);
+
+    // Ensures the notification header appears separate to the notification body text in high contrast mode
+    border-bottom: 1px solid transparent;
+
+    background-color: $govuk-brand-colour;
+
+    @include govuk-media-query($from: tablet) {
+      padding: (govuk-spacing(2) - $govuk-border-width) govuk-spacing(4) govuk-spacing(2);
```

Comment:
I think the vertical spacing on the header could be tighter so it looks a bit less chunky – I just tried `padding: 2px 20px 5px;` in my browser and it looks good to my eyes. Makes the coloured strip at the top about 40px tall in total including the top border. Not sure how that translates into the sass here 😬

Reply:
Also, at smaller sizes the `h2` sits a little too high in the coloured strip – I'd add `2px` to the top of it to make it more in the middle. Hopefully you can see the difference below. 

Before:
![Screenshot 2020-10-20 at 17 20 37](https://user-images.githubusercontent.com/19834460/96614979-a1c28000-12f8-11eb-85bb-c0b6b200363f.png)

After:
![Screenshot 2020-10-20 at 17 20 30](https://user-images.githubusercontent.com/19834460/96615036-b3a42300-12f8-11eb-9d95-eb4dd1738000.png)